src/style.css

```diff
@@ -1 +1,63 @@
 /* Styles go here */
+
+body {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  height: 100%;
```

Comment:
I don't think this works
Parent element does not have a height set, so this line does not work